lib/jekyll/assets/proxies/magick.rb

```diff
@@ -10,9 +10,7 @@
     @double @quadruple @half @one-third @two-thirds @one-fourth
       @two-fourths @three-fourths)
 
-  Jekyll::Assets::Env.liquid_proxies.add :magick, :img, *(args + presets) do
-    PRESETS = presets
-    ARGS = args
+  class JekyllAssetsMiniMagic
```

Comment:
Disclaimer: I'm not sure if changes I've made truly reflect what I intended to do, since I'm Ruby developer since like yesterday

Idea was to have feature like
```
{{ page.preview_image | asset_path : "magick:resize:600" }}
```
in addition to
```
{{ page.preview_image | img : "magick:resize:600" }}
```

So I've moved all actual processing logic to JekyllAssetsMiniMagic and adding that two times, one for :img and one for :asset_path
I've tried to look for similar code in other projects/plugins, yet without any particular luck